src/app/modules/form/admin-form/admin-form.middlewares.ts

```diff
@@ -26,5 +26,11 @@ export const updateSettingsValidator = celebrate({
       url: Joi.string().uri().allow(''),
       isRetryEnabled: Joi.boolean(),
     }).min(1),
+    endPage: Joi.object({
+      title: Joi.string(),
+      paragraph: Joi.string().allow(''),
+      buttonLink: Joi.string().uri().allow(''),
+      buttonText: Joi.string(),
+    }).min(1),
```

Comment:
hmm. so I think our internal mental model should still be that the endPage is separate from the rest of the settings. it's going to get very messy if the end page is editable both via its own endpoint and via the settings update endpoint.

we want the end page to be editable on both the settings page and the build tab but this is purely a UX enhancement. the way we think about the data model shouldn't change.